lib/tools/system-calls.js

```diff
@@ -884,13 +892,7 @@ systemCallMethods.root = async function root () {
  *                   if the switch failed.
  */
 systemCallMethods.unroot = async function unroot () {
-  try {
-    await exec(this.executable.path, ['unroot']);
-    return true;
-  } catch (err) {
-    log.warn(`Unable to unroot adb daemon: '${err.message}'. Continuing`);
-    return false;
-  }
+  await this.root(true);
```

Comment:
how about the returned value?

Reply:
Yep.... I really wish we used Typescript.